tool/README.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Publishing Devtool
+---
+
+* toc
+{:toc}
+
+# How to release the next version of DevTool.
```

Comment:
`DevTools`

nit: likely doesn't need to end in a period (and, do the other docs use H1 or H2 headers here? H2 is `##`)

Reply:
Right H2 thx.